### PR TITLE
Add MeuScrum to Project Management tools

### PR DESCRIPTION
**Pull request by silviodelgado**
MeuScrum is a free online tool for project management using Scrum framework.

### CODE REVIEW

**Comment by ripienaar** [comment]
thanks

File: README.md [hunk 1]
```diff
@@ -702,6 +702,7 @@ Table of Contents
    * [kanbantool.com](https://kanbantool.com/) — Kanban board-based project management. Free, paid plans with more options
    * [kanrails.com](https://kanrails.com/) — Kanban board-based project management. Free for 3 collaborators, 2 projects and 5 tracks. Paid plans available for unlimited collaborators, projects and tracks.
    * [LeanBoard](https://www.leanboard.io) — Collaborative whiteboard with sticky notes for your GitHub issues (Useful for Example Mapping and other techniques)
+   * [MeuScrum](https://www.meuscrum.com) - Free online scrum tool with kanban board
```
Comment (php-coder) [inline README.md]: Perhaps, it's better to use https://www.meuscrum.com/en Otherwise it opens in non-English localization by default.

Comment (ripienaar) [reply README.md]: weird, pretty sure when I checked it it was english default, that sounds good yeah, feel like a fix PR?

Comment (php-coder) [reply README.md]: I've just tested it again in a private windows in Firefox and it opens on Portuguese.

Comment (php-coder) [reply README.md]: I've submitted https://github.com/ripienaar/free-for-dev/pull/1454

Comment (silviodelgado) [reply README.md]: There are 3 localization options: 
https://www.meuscrum.com/en (english version)
https://www.meuscrum.com/pt (brazilian portuguese version)
https://www.meuscrum.com/es (spanish version)

https://www.meuscrum.com shows pt-br version by default.

**Pull request merged**